core/esp_spi.c

```diff
@@ -221,6 +221,26 @@ uint32_t spi_transfer_32(uint8_t bus, uint32_t data)
     return res;
 }
 
+static void _rearm_extras_bit(uint8_t bus, bool arm) {
+
+    if(!_minimal_pins[bus]) return ;
+    static uint8_t status = 0 ;
```

Comment:
ok i admit, missing this ^^.  I fix it without using one more byte. but  i dont no if it is better to save one byte or juste add one more like uint8_t status[bus].